backend/src/server.js

```diff
@@ -60,9 +61,28 @@ class CrudServer {
   }
 
   initMiddlewares() {
+    this.server.use(cookieParser());
+    this.server.use(passport.initialize());
+    this.server.use(passport.session());
```

Comment:
You don't need passport.session, deserializeUser, serializeUser at all

Reply:
ok

Reply:
this issue still not resolved. You still have `.serializeUser` and `.deserializeUser` in your code

Reply:
ok